### PR TITLE
Remove unneeded spaces for <b>, <u> + Refact

### PR DESCRIPTION
**Pull request by ivandrofly**
/cc @niksedk

### CODE REVIEW

File: src/Logic/Utilities.cs [hunk 1]
```diff
@@ -2972,16 +2971,15 @@ public static string RemoveUnneededSpaces(string text, string language)
             const string zeroWhiteSpace = "\u200B";
             const string zeroWidthNoBreakSpace = "\uFEFF";
             const string noBreakSpace = "\u00A0";
-            const string char160 = " "; // Convert.ToChar(160).ToString()
```
Comment (niksedk) [inline src/Logic/Utilities.cs]: Why did you remove this one?

Comment (ivandrofly) [inline src/Logic/Utilities.cs]: 160d = A0hex which is noBreakSpace
